### PR TITLE
[INLONG-503][Agent] The agent module specifies the indicator monitoring method

### LINKED ISSUE #503: [Improve][Doc] Modify the indicator monitoring method of the agent module
### Description

Modify the indicator monitoring method of the agent module

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Greedyu**
- Fixes #503 

### Motivation

Modify the indicator monitoring method of the agent module

### CODE REVIEW

File: docs/modules/agent/metrics.md [hunk 1]
```diff
@@ -15,16 +15,26 @@ Users can add similar JMX (port and authentication are adjusted according to the
...
-agent.prometheus.enable=true
-# the default is 8080
-agent.prometheus.exporter.port=8080
+# The listener of Prometheus is AgentPrometheusMetricListener
+agent.domainListeners=org.apache.inlong.agent.metrics.AgentPrometheusMetricListener
+# the default is 9080
+agent.prometheus.exporter.port=9080
 ```
 
+## User-Defined Configuration
```
Comment (dockerzhang) [inline docs/modules/agent/metrics.md:34]: User-Defined Configuration
->
Custome Configuration

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/agent/metrics.md [hunk 2]
```diff
@@ -14,17 +14,26 @@ Agent 提供了 JMX 和 Prometheus 方式的监控指标能力，默认使用 JM
...
-agent.prometheus.enable=true
-# 默认端口为8080
-agent.prometheus.exporter.port=8080
+# Prometheus监听类是AgentPrometheusMetricListener
+agent.domainListeners=org.apache.inlong.agent.metrics.AgentPrometheusMetricListener
+# 默认端口为9080
+agent.prometheus.exporter.port=9080
 ```
 
+## 自定义 配置
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/agent/metrics.md:33]: 自定义 配置
->
自定义配置

File: docs/modules/agent/metrics.md (hunk 1 above)
Comment (Greedyu) [reply docs/modules/agent/metrics.md]: Modified to Custom

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/agent/metrics.md (hunk 2 above)
Comment (Greedyu) [reply i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/agent/metrics.md]: resolve

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**